src/utils/cypressUtils.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * formatCypressId returns a string that can be used as a Cypress data ID
+ * selector (e.g. data-cy)
+ */
+export function formatCypressId(input: string): string {
+    // TODO: add unit tests when this package enables Jest
```

Comment:
Are maintainers open to me adding Jest to this package?

Reply:
sure

Reply:
Disregard, I thought Jest wasn't set up because I didn't see a jest config file. However, create-react-app includes this out of the box: https://create-react-app.dev/docs/running-tests

I'll push a commit to this PR with unit tests for this shortly